test/meson.build

```diff
@@ -16,6 +16,7 @@ tests = [
   'utils',
   'install',
   'service',
+  'status_file',
```

Comment:
Add this new file to https://github.com/rauc/rauc/blob/master/.github/workflows/sanitizers.yml, otherwise we loose address sanitizer coverage for these tests.

Reply:
Ack. Forgot that it is a separate list. Will add.